modules/widgets/contact-info.php

```diff
@@ -33,8 +34,21 @@ function __construct() {
 				$widget_ops
 			);
 			$this->alt_option_name = 'widget_contact_info';
+
+			if ( is_customize_preview() ) {
+				add_action( 'wp_enqueue_scripts', array( $this, 'enqueue_scripts' ) );
```

Comment:
Hi @westonruter! is this really necessary? The scripts are enqueued when the [map is built](https://github.com/Automattic/jetpack/pull/3543/files#diff-2fbcbb8ad1fe6e0bf7a8c111cd17adfaR261). Maybe I'm missing something because I think this might actually be disadvantageous if we do this for all widgets. We will end up with a lot of JS and CSS loaded in Customizer whether they're in use by a widget or not which will add to the Customizer loading times.
If possible, I'd love to talk for a bit in WordPress Slack whenever it's possible to you.
Cheers!